src/test/java/org/spideruci/tacoco/util/PathBuilderTest.java

```diff
@@ -0,0 +1,45 @@
+package org.spideruci.tacoco.util;
```

Comment:
I love tests! But what do these tests really mean? test, test1 test2 are non-informative names that tell me nothing when the tests actually fail. just something to note for future testing in this or in any other project. :)